apps/trading-e2e/src/integration/market.cy.ts

```diff
@@ -0,0 +1,163 @@
+          cy.getByTestId(itemHeader).should('have.text', 'Volume');
+          cy.getByTestId(itemValue).should('not.be.empty');
+        });
+      });
+    });
+
+    it('must see market mode', () => {
+      cy.getByTestId(marketSummaryBlock).within(() => {
+        cy.getByTestId(marketMode).within(() => {
+          cy.getByTestId(itemHeader).should('have.text', 'Trading mode');
+          cy.getByTestId(itemValue).should('not.be.empty');
+        });
+      });
+    });
+
+    it('must see market settlement', () => {
+      cy.getByTestId(marketSummaryBlock).within(() => {
+        cy.getByTestId(marketSettlement).within(() => {
+          cy.getByTestId(itemHeader).should('have.text', 'Settlement asset');
+          cy.getByTestId(itemValue).should('not.be.empty');
+        });
+      });
+    });
+  });
+
+  describe('Market tooltips', { tags: '@regression' }, () => {
+    it('should see expiry tooltip', () => {
+      cy.getByTestId(marketSummaryBlock).within(() => {
+        cy.getByTestId(marketExpiry).within(() => {
+          cy.getByTestId(itemValue)
+            .should('have.text', 'Not time-based')
+            .realHover();
+        });
+      });
+      cy.getByTestId('expiry-tool-tip')
+        .should(
+          'contain.text',
+          'This market expires when triggered by its oracle, not on a set date.'
+        )
+        .within(() => {
+          cy.getByTestId('link')
+            .should('have.attr', 'href')
+            .and('include', 'https://explorer.fairground.wtf/');
+        });
+    });
+
+    it('should see trading conditions tooltip', () => {
+      const toolTipLabel = 'tooltip-label';
+      const toolTipValue = 'tooltip-value';
+      const auctionToolTipLabels = [
+        'Auction start',
+        'Est auction end',
+        'Target liquidity',
+        'Current liquidity',
+        'Est uncrossing price',
+        'Est uncrossing vol',
+      ];
+
+      cy.getByTestId(marketSummaryBlock).within(() => {
+        cy.getByTestId(marketMode).within(() => {
+          cy.getByTestId(itemValue)
+            .should('contain.text', 'Monitoring auction')
+            .and('contain.text', 'liquidity')
+            .realHover();
+        });
+      });
+
+      cy.getByTestId('trading-mode-tooltip')
+        .should(
+          'contain.text',
+          'This market is in auction until it reaches sufficient liquidity.'
+        )
+        .within(() => {
+          cy.getByTestId('external-link')
+            .should('have.attr', 'href')
+            .and(
+              'include',
+              'https://docs.vega.xyz/docs/testnet/concepts/trading-on-vega/trading-modes#auction-type-liquidity-monitoring'
+            );
+
+          for (let i = 0; i < 6; i++) {
+            cy.getByTestId(toolTipLabel)
+              .eq(i)
+              .should('have.text', auctionToolTipLabels[i]);
+            cy.getByTestId(toolTipValue).eq(i).should('not.be.empty');
+          }
+        });
+    });
+  });
+});
```

Comment:
I would maybe name this file market-summary.cy.ts?  Just so it's bit more differentiated from markets.cy.ts

Reply:
I just followed the way of naming that we have in types and queries (e.g. Market, Markets, MarketData, MarketsData), but I think you're right. Changing.